core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -447,7 +447,7 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
           fail("No message should be sent successfully.")
         } catch {
           case e: Exception =>
-            assertEquals("java.lang.IllegalStateException: Producer is closed forcefully.", e.getMessage)
+            assertEquals("org.apache.kafka.common.KafkaException: Producer is closed forcefully.", e.getMessage)
```

Comment:
I know it is from the original code, but asserting the error message seems dubious. Maybe we can just verify the exception type is `KafkaException`?